### DISCUSSION TITLE
bark.cpp? New TTS model released

### DISCUSSION THREAD

**Discussion by r3nor** [ggml-org/llama.cpp#1089] [Ideas]
Bark is a transformer-based text-to-audio model created by [Suno](https://suno.ai/). Bark can generate highly realistic, multilingual speech as well as other audio – including music, background noise and simple sound effects. The model can also produce nonverbal communications like laughing, sighing and crying. They are providing access to pretrained model checkpoints ready for inference.

Repo: https://github.com/suno-ai/bark
Examples: https://suno-ai.notion.site/Bark-Examples-5edae8b02a604b54a42244ba45ebc2e2

It would be awesome to have a CPP version of Bark :)

**Comment by ggerganov** [comment]
Looks like a great TTS model to implement with `ggml`.
After we finish with the quantization improvements, I think we can take a detailed look and try to support it

**Reply by r3nor** [reply]
That would be great!

**Reply by ehartford** [reply]
Yes Please I will be very happy to use this.

**Reply by qnixsynapse** [reply]
um. Any updates?

**Reply by monatis** [reply]
@PABannier has an ongoing effort at  https://github.com/PABannier/bark.cpp

**Comment by Green-Sky** [comment]
gonna link this for discoverability
https://github.com/ggerganov/ggml/issues/59

**Comment by mirek190** [comment]
I just tested it ... IS INSAME GOOD... just wow.   Can you imagine next iterations ...
